tests/e2e/api/controllers/_changes.spec.js

```diff
@@ -10,6 +10,7 @@ const DEFAULT_EXPECTED = [
   'appcache',
   'settings',
   'resources',
+  'branding',
```

Comment:
This adds a test to ensure all users get the branding doc.

Reply:
I see